tests/test_CompareEclipseRestart.cpp

```diff
@@ -0,0 +1,173 @@
+#include "test_CompareEclipseRestart.hpp"
+
+int main(int argc, char** argv) {
+    char* gridFile1 = argv[1];
+    char* gridFile2 = argv[2];
+    char* unrstFile1 = argv[3];
+    char* unrstFile2 = argv[4];
+
+    if (argc != 5) {
+        printHelp();
+    }
+    else {
+        // Comparing grid sizes from .EGRID-files
+        {
+            ecl_grid_type* ecl_grid1 = ecl_grid_alloc(gridFile1);
```

Comment:
Prefer `const` for variables you don't need to change, makes it a lot easier for humans and compilers to understand the intentions of the code. I.e. `const ecl_grid_type*` here, `const int gridCount1` below etc.